modules/salt/pages/salt-rate-limiting.adoc

```diff
@@ -0,0 +1,60 @@
+[[salt.rate.limiting]]
+= Rate Limiting
+
+Salt is able to run commands in parallel on a large number of minions.
+This can potentially create large amounts of load on your infrastructure.
+You can use these rate-limiting parameters to control the load in your environment.
+
+These parameters are all configured in the [filename]``/etc/rhn/rhn.conf`` configuration file.
+
+[NOTE]
+====
+Salt commands that are executed from the command line are not subject to these parameters.
+====
+
+
+
+== Batch Size
+
+This parameter sets the maximum number of clients that can execute a single action at the same time.
+
+Choosing a lower number will reduce system load and parallelism, but might reduce overall performance for processing actions.
+
+Adjust the [systemitem]``java.salt_batch_size`` parameter.
+Defaults to 100.
+
+
+
+== Batch Delay
+
+This parameter sets the amount of time, in seconds, to wait after an action is processed before beginning to process the action on the next minion.
+To begin, actions are processed on the number of minions determined in the batch size parameter.
+As each minion completes, the action begins on a new minion after the specified delay period.
+
+Increasing the delay increases the chance that multiple minions will have completed before the next action is issued, resulting in fewer overall commands, and reducing load.
+
+Adjust the [systemitem]``java.salt_batch_delay`` parameter.
+Defaults to 1.0 seconds.
+
+
+
+== Ping Timeout
+
+During Salt batch calls, Salt clients will perform a presence ping.
```

Comment:
Maybe we could say "At the beginning" instead of "During", given that that's the way it works.

Reply:
OK, what about  this?

```suggestion
Salt batch calls begin with the Salt master performing a presence ping on the minions.
```

Reply:
Nice, thanks